src/QueryM/Crypto.purs

```diff
@@ -0,0 +1,135 @@
+module QueryM.Crypto
+  ( hashData
+  , hashScript
+  , datumHash
+  , plutusHash
+  , HashedData(..)
+  , HashMethod(..)
+  ) where
+
+import Prelude
+
+import Affjax as Affjax
+import Affjax.RequestBody as Affjax.RequestBody
+import Affjax.ResponseFormat as Affjax.ResponseFormat
+import Control.Monad.Reader.Trans (asks)
+import Data.Argonaut
+  ( class DecodeJson
+  , Json
+  , JsonDecodeError(..)
```

Comment:
We also use explicit constructor imports

Reply:
I have to configure my lsp properly at some point, it always introduces those :/